src/css/components/help-menu.less

```diff
@@ -0,0 +1,42 @@
+/* =============
+ * Help Menu
+ * =============
+ */
+.help-menu {
+  float: left;
+  margin: @base-spacing-unit * 2.5 @base-spacing-unit;
+  width: @base-spacing-unit * 2.5;
```

Comment:
Yeah!
But we write it like this:
`width: @base-spacing-unit*2.5;`
to differentiate at such `margin: @base-spacing-unit*2.5 @base-spacing-unit*5`